### DISCUSSION TITLE
JW.org in M³ opens new browser tabs

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#770] [Q&A] [answered]
It looks very good this feature, now how to deal when the site opens a new tab?

**Comment by mtdvlpr** [comment, marked as answer]
@e-Feitosa, I see what you mean! I have fixed the issue and in the next release links that would open a new tab are now opened in M³ as you would expect. For now, just close the tabs and don't click on links that take you to another site (online library, jw hub, etc.)
